src/pages/dashboard/panels/ShokoNews.tsx

```diff
@@ -9,9 +9,21 @@ import { DashboardNewsType } from '@/core/types/api/dashboard';
 
 import { useGetShokoNewsFeedQuery } from '@/core/rtkQuery/externalApi';
 
+const newNewsCheck = (date: string) => {
+  const itemDate = new Date(date);
+  const currentDate = new Date();
+  const oneDayInMilliseconds = 24 * 60 * 60 * 1000;
+  const differenceInMilliseconds = currentDate.getTime() - itemDate.getTime();
+  const differenceInDays = Math.floor(differenceInMilliseconds / oneDayInMilliseconds);
+  return differenceInDays <= 14;
+};
+
 const NewsRow = ({ item }: { item: DashboardNewsType }) => (
   <div className="flex flex-col" key={item.title}>
-    <p className="font-semibold">{item.date_published}</p>
+    <div className="flex gap-x-4">
```

Comment:
Just a little thing for the future. This is still fine, you can leave it.
As both the child elements have the class `font-semibold`, you could have just added the class here for the div and removed them from the children